ktor-client/ktor-client-core/common/src/io/ktor/client/plugins/observer/DelegatedCall.kt

```diff
@@ -40,6 +40,16 @@ public fun HttpClientCall.wrap(content: ByteReadChannel, headers: Headers): Http
     return DelegatedCall(client, content, this, headers)
 }
 
+/**
+ * Wrap existing [HttpClientCall] with new [headers] and content produced by the given [block].
+ *
+ * [Report a problem](https://ktor.io/feedback/?fqname=io.ktor.client.plugins.observer.wrap)
+ */
+@InternalAPI
+public fun HttpClientCall.wrap(headers: Headers, block: () -> ByteReadChannel): HttpClientCall {
```

Comment:
could we make it internal for the patch release?

Reply:
Rebased onto EAP branch